samples/deploy/crd/v1/ApisixConsumer.yaml

```diff
@@ -108,6 +109,12 @@ spec:
                         value:
                           type: object
                           properties:
+                            server:
```

Comment:
Why update here?

Reply:
Sorry, Didn't pay attention when resolving the conflict.